### ISSUE TITLE
Team TA - Week 4 : Split the data in validation, train, test and train your model

### ISSUE THREAD

**Issue by uroojbadar** [CubeStatistica/Professional-Training-for-Data-Science-C1#39]
1.  Split the data into 75% training set, 10% validation set, and  15% test set.
2. Prepare your data for modelling by turning all the categorical features into numeric ones.
3. Fit the Xg boost model on your trained data. Use default parameters for training.

**Comment by uroojbadar** [comment]
- [ ] @Zobia12 
- [ ] @SuneetaDevi 
- [ ] @uroojbadar
